packages/mdc-select/_mixins.scss

```diff
@@ -212,3 +225,173 @@
+    @include mdc-floating-label-shake-animation(select-outlined-leading-icon);
+
+    @include mdc-rtl {
+      @include mdc-floating-label-shake-animation(select-outlined-leading-icon-rtl);
+    }
+  }
+
+  &.mdc-select__menu .mdc-list-item__text {
+    @include mdc-rtl-reflexive-property(padding, 32px /* left-value */, 32px /* right-value */);
+  }
+}
+
+@mixin mdc-select-text-and-control_ {
+  .mdc-select__selected-text {
+    min-width: 200px;
+    padding-top: 22px;
+  }
+
+  .mdc-select__native-control,
+  .mdc-select__selected-text {
+    @include mdc-rtl-reflexive-property(padding, $mdc-select-label-padding, $mdc-select-arrow-padding);
+    @include mdc-typography(subtitle1);
+
+    &::-ms-expand {
+      display: none;
+    }
+
+    &::-ms-value {
+      background-color: transparent;
+      color: inherit;
+    }
+
+    // counteracts the extra text padding that Firefox adds by default
+    @-moz-document url-prefix("") {
+      text-indent: -2px;
+    }
+
+    > option {
+      @include mdc-theme-prop(background-color, surface);
+
+      color: inherit; // Override default user agent stylesheet
+    }
+
+    box-sizing: border-box;
+    width: 100%;
+    height: $mdc-select-height;
+    padding-top: 20px;
+    padding-bottom: 4px;
+    border: none;
+    border-bottom: 1px solid;
+    outline: none;
+    background-color: transparent;
+    color: inherit; // Override default user agent stylesheet
+    white-space: nowrap;
+    cursor: pointer;
+    appearance: none;
+  }
+}
+
+@mixin mdc-select-disabled_ {
+  @include mdc-select-container-fill-color_($mdc-select-disabled-fill-color);
+
+  .mdc-floating-label {
+    @include mdc-floating-label-ink-color($mdc-select-disabled-label-color);
+  }
+
+  .mdc-select__dropdown-icon {
+    @include mdc-select-dd-arrow-svg-bg_($mdc-select-dropdown-color, $mdc-select-disabled-dropdown-opacity);
+  }
+
+  .mdc-line-ripple {
+    display: none;
+  }
+
+  .mdc-select__icon {
+    @include mdc-theme-prop(color, $mdc-select-disabled-icon-color);
+  }
+
+  .mdc-select__native-control,
+  .mdc-select__selected-text {
+    @include mdc-theme-prop(color, $mdc-select-disabled-ink-color);
+
+    border-bottom-style: dotted;
+  }
+
+  .mdc-select__selected-text {
+    pointer-events: none;
+  }
+
+  &.mdc-select--outlined {
+    @include mdc-select-container-fill-color_(transparent);
+
+    .mdc-select__native-control,
+    .mdc-select__selected-text {
+      border-bottom-style: none;
+    }
+
+    @include mdc-select-outline-color_($mdc-select-outlined-disabled-border);
+  }
+
+  cursor: default;
+  pointer-events: none;
+}
+
+@mixin mdc-select-outlined_ {
+  @include mdc-select-container-fill-color(transparent);
+  @include mdc-select-outline-color($mdc-select-outlined-idle-border);
+  @include mdc-select-hover-outline-color($mdc-select-outlined-hover-border);
+  @include mdc-select-focused-outline-color(primary);
+  @include mdc-floating-label-float-position($mdc-select-outlined-label-position-y);
```

Comment:
I think this line is now redundant since mdc-select-outline-shape-radius invokes this a few lines later. This line was removed in #3929 (and was one of the conflicts listed earlier, I think).